src/contentscript/index.ts

```diff
@@ -4,10 +4,25 @@ interface MessageWithResponse {
     name: string;
 }
 
+function hoverDecorator(event: MouseEvent) {
+    const hoveredElement = <HTMLElement>event.target;
+    const style = hoveredElement.style.outline;
+    hoveredElement.style.outline = 'yellow dashed 5px';
```

Comment:
hoveredElement.classList.add(customClass)

https://stackoverflow.com/questions/1720320/how-to-dynamically-create-css-class-in-javascript-and-apply